README.md

```diff
@@ -66,6 +66,8 @@ cargo build
 docker-compose up -d db s3
 # Setup the database you just created
 cargo run -- database migrate
+# To be noted: if you want to revert to a precise migration, you can run:
```

Comment:
I would put this down by the 'database' section, not right up front in the getting started section: https://github.com/rust-lang/docs.rs/blob/abd7e4d86d45c168a5672feacbc1b8ee8f470423/README.md#database-subcommand
Also I think you can remove 'To be noted', the rest stands fine on its own IMO.